README.md

```diff
@@ -27,43 +35,41 @@ Here's a simple example to get you started.
 </select>
 <button id="purchase">Buy Button</button>
 
-<script
-    type="text/javascript"
-    src="https://checkout.freemius.com/js/v1/"
-></script>
-
-<script type="text/javascript">
-    const handler = new FS.Checkout({
-        plugin_id: '9885',
-        plan_id: '16634',
-        public_key: 'pk_ccca7be7fa43aec791448b43c6266',
-        image: 'https://your-plugin-site.com/logo-100x100.png',
-    });
+<script type="module">
```

Comment:
The thing with `script` type `module` is that it will look for imported modules with a HTTP request ([doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)). So installing with a package manager like `npm` won't really do anything.

So I suggest to simply remove the `<script type="module">` from the example and perhaps write something like

```md
## Usage Guide

Here's a simple example to get you started. This example assumes you're using some bundler like vite or webpack to manage your app.

...

Finally in your JS app, you can do:

```js
import { Checkout } from '@freemius/checkout';
...
```